src/components/general/Slider/Slider.stories.tsx

```diff
@@ -28,7 +28,12 @@ const SliderPercentageStory = () => {
 
     return (
         <div style={{ width: 400, margin: '0 auto' }}>
-            <Slider value={value} markers={markers} onChange={marker => setValue(marker.value)} />
+            <Slider
+                value={value}
+                disabled

```

Comment:
Do we want the slider in the story to be disabled?

Reply:
We now have two sliders in the story one that is disabled and one active